packages/lib-core/src/types/window.ts

```diff
@@ -0,0 +1,11 @@
+import type { compose } from 'redux';
+
+declare global {
+  interface Window {
+    SERVER_FLAGS: {
+      basePath: string;
+    };
```

Comment:
We won't have a server, so this is not going to be needed.

Reply:
I actually like this idea... we are the middle ground package, we don't need to come with this support.

Reply:
We may want to expose things later for testing scenarios, but for now I think we should avoid it.